pkg/pgclient/client.go

```diff
@@ -30,17 +30,17 @@ type Config struct {
 
 // ParseFlags parses the configuration flags specific to PostgreSQL and TimescaleDB
 func ParseFlags(cfg *Config) *Config {
-	flag.StringVar(&cfg.host, "pg-host", "localhost", "The PostgreSQL host")
-	flag.IntVar(&cfg.port, "pg-port", 5432, "The PostgreSQL port")
-	flag.StringVar(&cfg.user, "pg-user", "postgres", "The PostgreSQL user")
-	flag.StringVar(&cfg.password, "pg-password", "", "The PostgreSQL password")
-	flag.StringVar(&cfg.database, "pg-database", "postgres", "The PostgreSQL database")
-	flag.StringVar(&cfg.sslMode, "pg-ssl-mode", "disable", "The PostgreSQL connection ssl mode")
-	flag.IntVar(&cfg.dbConnectRetries, "pg-db-connect-retries", 0, "How many times to retry connecting to the database")
+	flag.StringVar(&cfg.host, "db-host", "localhost", "The TimescaleDB host")
```

Comment:
i'm not sure about this change, I wanted to signify that the connector is no longer compatible with PostgreSQL, but it is still a Postgres cluster with users, databases etc created in Postgres

Reply:
I like this change